ansible/roles/webserver-content/tasks/main.yml

```diff
@@ -70,3 +82,33 @@
     block: |
       www-data ALL=(ALL) ALL
       www-data ALL=(ALL) NOPASSWD: /usr/bin/pumount
+
+- name: checkout NextCloud integration
+  git:
+   repo: https://github.com/adavenpo/BibleBox-python-app.git
+   dest: "/tmp/BibleBox-nextcloud"
+   depth: 1
+   #force: yes
+
+- name: checkout latest owncloud lib
+  git:
+    repo: https://github.com/owncloud/pyocclient.git
+    dest: /tmp/pyocclient
+    depth: 1
+    #force=yes
+
+- name: Pull python packages
+  shell: "pip install webapp2 webob"
+
+- name: deploy NextCloud integration static content
+  shell: "cp -a /tmp/BibleBox-nextcloud/static/* {{ biblebox_default_content_root }}"
+
+- name: deploy NextCloud integration WSGI app
+  shell: "cp -a /tmp/BibleBox-nextcloud/wsgi /var/www/wsgi"
+
+- name: Add app link to uwsgi config
+  shell: "ln -fs /var/www/wsgi/app.ini /etc/uwsgi/apps-enabled/app.ini"
```

Comment:
Please use the [file task](http://docs.ansible.com/ansible/file_module.html) to create the symlink.